iOS/Flows/Conversation/Cells/MessagesTimeMachineCollectionViewLayout.swift

```diff
@@ -254,13 +266,46 @@ class MessagesTimeMachineCollectionViewLayout: TimeMachineCollectionViewLayout {
         return CGPoint(x: proposedContentOffset.x, y: proposedContentOffset.y + self.scrollOffset)
     }
 
+    /// NOTE: Disappearing does not mean that the item will not be visible after the animation.
+    /// Per the docs:  "For each element on screen before the invalidation, finalLayoutAttributesForDisappearingXXX will be called..."
```

Comment:
The apple documentation online is not correct. Comments in the header files are though. "Disappearing" doesn't actually mean it's disappearing, just that it was visible before the animation.